hudi-common/src/main/java/org/apache/hudi/common/model/HoodiePartitionMetadata.java

```diff
@@ -38,6 +38,7 @@
   public static final String HOODIE_PARTITION_METAFILE = ".hoodie_partition_metadata";
   public static final String PARTITION_DEPTH_KEY = "partitionDepth";
   public static final String COMMIT_TIME_KEY = "commitTime";
+  public static final String ACTION_TYPE_KEY = "actionType";
```

Comment:
Can you explain why we need this change ?

Reply:
I thought having all metadata constants in one place would make it simpler. This is used in reading archived commit. I can move the constant to ArchivedTimeline if you think thats a better place.

Reply:
Please move it to the ArchivedTimeline since it makes more senes there

Reply:
sounds good. I'm sending an update